libs/obs/include/mrpt/obs/CObservation3DRangeScan.h

```diff
@@ -47,12 +47,16 @@ struct T3DPointsProjectionParams
 	/** (Default:true) set to false if you want to preserve the organization of
 	 * the point cloud */
 	bool MAKE_DENSE;
+	/** (Default:false) set to true if you want to use an organized point cloud */
+	bool MAKE_ORGANIZED;
+
 	T3DPointsProjectionParams()
 		: takeIntoAccountSensorPoseOnRobot(false),
 		  robotPoseInTheWorld(nullptr),
 		  PROJ3D_USE_LUT(true),
 		  USE_SSE2(true),
-		  MAKE_DENSE(true)
+		  MAKE_DENSE(true),
+		  MAKE_ORGANIZED(false)
```

Comment:
good!

Reply:
Forgot to add this,
MAKE_DENSE does not actually preserve the "organization" of the point cloud. It is only used to control whether or not the points can have nan / undefined values. If MAKE_DENSE is true, the cloud simply cannot contain undefined points and all the points contain finite values.